### PR TITLE
feat: add pipeline file and templates

### PR DESCRIPTION
**Pull request by mapleknight**

### CODE REVIEW

File: deploy/settings.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'a=()q#x6$s_2tnzgvoudd=5&gbv*57bcr(^%f)&rp-7ydf(9ao'
```
Comment (southalln) [inline deploy/settings.py:23]: Not sure this should be committed into master

File: deploy/settings.py [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = ['ALLOWED_HOSTS_VALUE']
+
+
+# Application definition
+
+INSTALLED_APPS = [
```
Comment (southalln) [inline deploy/settings.py:33]: Can you import settings.py from https://github.com/mapleknight/Relay/blob/master/tr_sys/tr_sys/settings.py and only update the variables that you need?

File: deploy/settings.py (hunk 1 above)
Comment (mapleknight) [reply deploy/settings.py:23]: I will update it to a placeholder and replace it during deployment similar to DB credentials

File: deploy/settings.py (hunk 2 above)
Comment (mapleknight) [reply deploy/settings.py:33]: @southalln The settings.py in deploy folder is a template file.If we can use this template file to replace the tr_sys/tr_sys/settings.py, we can keep only one copy of it.

**Review by southalln — APPROVED** [review]

**Pull request merged**